data/Aeneid/Aeneid01.textproto

```diff
@@ -1,4 +1,4 @@
-name: "Aen. 1"
```

Comment:
Did you manually change this? I think when I ran latin_scan, it changed it back to the original.

Reply:
I set it via `--name`, yes.

Reply:
BTW, you can set this however you want so long as we're consistent across the 6 files. In fact we don't have to set it at all, it's optional and we could omit the line altogether.